test/org/javarosa/core/benchmark/FormParserHelperParseISIBenchmark.java

```diff
@@ -0,0 +1,59 @@
+package org.javarosa.core.benchmark;
+
+import org.javarosa.core.model.FormDef;
+import org.javarosa.core.reference.ReferenceManagerTestUtils;
+import org.javarosa.core.util.PathConst;
+import org.javarosa.xform.parse.FormParserHelper;
+import org.junit.Test;
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.Level;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.State;
+import org.openjdk.jmh.infra.Blackhole;
+import org.openjdk.jmh.results.RunResult;
+import org.openjdk.jmh.runner.NoBenchmarksException;
+import org.openjdk.jmh.runner.Runner;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.nio.file.Path;
+
+import static org.javarosa.test.utils.ResourcePathHelper.r;
+
+public class FormParserHelperParseISIBenchmark {
```

Comment:
Don't use abbreviations, especially when they hide key aspects of the benchmark i.e. the fact that this benchmark is measuring *internal* secondary instances.